pkg/api/customization/cluster/actionhandler.go

```diff
@@ -293,8 +295,17 @@ func (a ActionHandler) viewMonitoring(actionName string, action *types.Action, a
 		return httperror.NewAPIError(httperror.InvalidState, "disabling Monitoring")
 	}
 
+	appName, appTargetNamespace := monitoring.ClusterMonitoringInfo()
+	appList, err := a.AppClient.List(monitoring.OwnedAppListOptions(cluster.Name, appName, appTargetNamespace))
```

Comment:
Can we use `Client.Get` instead of `List`? I think we know the exact app name for cluster monitoring app.

Reply:
If using `Client.Get`, we need to do something like `monitoring.GetSystemProjectID` to get the `System` project ID, as the cluster-level Monitoring apps are deploying in the `System` project.